fdbserver/OldTLogServer_6_2.actor.cpp

```diff
@@ -339,7 +339,7 @@ struct TLogData : NonCopyable {
 	int64_t targetVolatileBytes; // The number of bytes of mutations this TLog should hold in memory before spilling.
 	int64_t overheadBytesInput;
 	int64_t overheadBytesDurable;
-	int activePeekStreams;
+	int activePeekStreams = 0;
```

Comment:
it seems like it would be slightly cleaner to initialize this where all of the other counters are being initialized, in the constructor starting on line 364.

Reply:
I understand it's good to keep consistency with the old way how FDB initializes counters. But I still think since c++11 we should leverage the strength of initialization in the same line... At least in the future, we don't need to change every constructor places when adding some members with default initial value : )

Reply:
Ok, that's fair.